atom/browser/net/atom_url_request.cc

```diff
@@ -121,6 +124,9 @@ void AtomURLRequest::DoInitialize(
   request_->set_method(method);
   // Do not send cookies from the cookie store.
   DoSetLoadFlags(net::LOAD_DO_NOT_SEND_COOKIES);
+  // Set a flag to stop custom protocol from intercepting this request.
+  request_->SetUserData(api::DisableProtocolInterceptFlagKey(),
```

Comment:
This can just be `request_->SetUserData(key, base::MakeUnique<base::SupportsUserData::Data>())` , can avid declaring `class DisableProtocolInterceptFlag`

Reply:
:+1: